btcnode.adoc

```diff
@@ -0,0 +1,95 @@
+= Bitcoin Core Nodes
+
+Bisq contributors run a federation of Bitcoin Core nodes to ensure Bisq connects to nodes with consistent Bitcoin Core implementations.
```

Comment:
I would add here the bloom filer/privacy reason as that is a very strong reason.